jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/test/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsTabPresenterTest.java

```diff
@@ -111,6 +115,15 @@ public void setProcessInstanceDetailsTest() {
                           nodeInstanceSummary.getType());
     }
 
+    @Test
+    public void setProcessInstanceDetailsTestWihtoutParentProcessInstance() {
+        when(processRuntimeDataServiceMock.getProcessInstance(any())).thenReturn(null);
+        presenter.setProcessInstance(processInstanceSummary);
+
+        verify(view, times(1)).setParentProcessInstanceIdText("", false);
```

Comment:
times(1) is the default anyway, you dont need to set.
```suggestion
        verify(view).setParentProcessInstanceIdText("", false);
```

Reply:
Done